src/libraries/Common/src/Interop/Linux/cgroups/Interop.cgroups.cs

```diff
@@ -116,6 +209,7 @@ private static CGroupVersion FindCGroupVersion()
                 {
                     "cgroup2fs" => CGroupVersion.CGroup2,
                     "tmpfs" => CGroupVersion.CGroup1,
+                    "udev" => CGroupVersion.CGroup1,
```

Comment:
This is a hotfix for a bug in cgroup v1 handling (https://github.com/dotnet/runtime/issues/93581)

Reply:
Is this fix reliable? From #93581:

> The reason is that we get the format string from the UnixFileSystemTypes enum based on the filesystem "magic" constant and the udev and tmpfs (and few others) share the same value.

It means that the `DriveInfo.DriveFormat` is going to return one of the strings at random. It can start returning yet another string that happens to match the magic value tomorrow.

I think there are two problems here:
-  `DriveInfo.DriveFormat` should be more deterministic
- We probably should not be using `DriveInfo.DriveFormat` in the first place. It is unnecessarily complex API for what we need to do here. Can we call the relevant System.Native shim API directly here?

Reply:
I have actually put this change in originally just to enable me to test that it works correctly in cgroups v1 modulo this bug. I was going to leave the real fix for a separate PR and actually probably remove this hotfix from this PR. I agree that calling the relevant System.Native shim API directly here sounds like the right fix.